src/main/resources/templates/polls/detail.twig

```diff
@@ -4,28 +4,32 @@
 {% block description %}{{ poll.title }} {{ poll.options }}{% endblock %}
 
 {% block content %}
-	<h1>{{ poll.name }}</h1>
+	<h1>{{ poll.title }}</h1>
 
-	<form method="post" action="/polls/vote/{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}" class="poll">
-		{%- for option, nVotes in poll.options %}
+	<div class="poll">
```

Comment:
I'd use another class for this. The poll class will get polluted. .vote might be a better choice